C/c4Private.h

```diff
@@ -55,6 +55,17 @@ bool c4log_getWarnOnErrors(void) C4API;
     exception occurs. */
 void c4log_enableFatalExceptionBacktrace(void) C4API;
 
+typedef C4_ENUM(uint32_t, C4DatabaseTag) {
+    DatabaseTagAppOpened,
+    DatabaseTagReplicator,
+    DatabaseTagBgDB,
+    DatabaseTagREST,
+    DatabaseTagOther
```

Comment:
I understand that the previous PR was using the other tag as the default tag for internal database open/reopen API. But as we have changed the API for setting the tag, do we still need the Other tag? I don't see it used anywhere as well.

Reply:
Yes, you're right. We reversed the logic in this regard. Now I don't need it. (In the first version, DataFile gets default internal tag, DatabaseTagOther. We reversed it and it takes AppOpened tag by default.)  Going to remove it.